python/lsst/pipe/tasks/calibrate.py

```diff
@@ -40,6 +40,8 @@
 from lsst.pipe.tasks.setPrimaryFlags import SetPrimaryFlagsTask
 from .fakes import BaseFakeSourcesTask
 from .photoCal import PhotoCalTask
+from .computeExposureSummaryStats import ComputeExposureSummaryStatsTask
+
```

Comment:
Extra newline?  Although...looking at https://developer.lsst.io/python/style.html?highlight=__all__#standard-code-order-should-be-followed, the `__all__` is in the wrong place (but I think this is pervasive...and certainly not on you on this tickte!)

Reply:
There are supposed to be two blank lines between the imports and the `__all__`.